test/test_guestbook_go.rb

```diff
@@ -113,7 +113,7 @@ def delete_replication_controllers(client, namespace, replication_controllers)
   private
 
   def construct_base_rc(namespace)
-    rc = Kubeclient::ReplicationController.new
+    rc = Kubeclient::Resource.new
```

Comment:
Shouldn't one set `rc.kind` now that it's not implied by class?
IIUC, it works because we use these objects via `create_replication_controller`.
It's not ideal end state (though maybe fine for now) if some Resource objects around will have `.kind` and some won't, and what works depends on how you use it...
cf #235, #241.

Reply:
It is correct though it did not change in this PR.